nautobot/extras/api/serializers.py

```diff
@@ -918,7 +949,7 @@ def validate(self, data):
 class JobInputSerializer(serializers.Serializer):
     data = serializers.JSONField(required=False, default=dict)
     commit = serializers.BooleanField(required=False, default=None)
-    scheduled_job = NestedScheduledJobSerializer(required=False)
+    schedule = NestedScheduledJobCreationSerializer(required=False)
```

Comment:
Here and in `extras/api/views.py` and in `extras/tests/test_api.py` I'm reverting this serializer field from `scheduled_job` (change made in #3262) back to `schedule`; as discussed, this isn't actually a reference to a ScheduledJob instance, it's a schedule for the job that may or may not result in the creation of a ScheduledJob instance, so I think `schedule` is more accurate.